### PR TITLE
add GitHub actions state to README

### PR DESCRIPTION
**Pull request by kerolloz**
image

### CODE REVIEW

**Comment by dylanmtaylor** [comment]
Oooh, this is awesome. Great to see a badge showing the CI works

**Comment by dylanmtaylor** [comment]
So the one thing I'd fix is that https:// link being on the readme if that's how it looks in GitHub, as seen in your screenshot

File: README.md [hunk 1]
```diff
@@ -7,9 +7,11 @@
   <img src="https://img.shields.io/github/license/ventoy/Ventoy?style=for-the-badge">
   <img src="https://img.shields.io/github/stars/ventoy/Ventoy?style=for-the-badge">
   <img src="https://img.shields.io/github/downloads/ventoy/Ventoy/total.svg?style=for-the-badge">
+  <img src="https://img.shields.io/github/workflow/status/ventoy/Ventoy/Ventoy%20CI?label=actions&logo=github&style=for-the-badge">
+  
 </p>
 
-<h4 align="left">
+<h4 align="left">https://img.shields.io/github/workflow/status/ventoy/Ventoy/Ventoy%20CI?label=actions&logo=github&style=for-the-badge
```
Comment (dylanmtaylor) [inline README.md:14]: Remove https://img.shields.io/github/workflow/status/ventoy/Ventoy/Ventoy%20CI?label=actions&logo=github&style=for-the-badge from this line

Comment (kerolloz) [reply README.md:14]: oh sorry.. a copy and paste mistake

Comment (kerolloz) [reply README.md]: done

**Comment by kerolloz** [comment]
image

**Pull request merged**